engine/uState.mli

```diff
@@ -23,25 +23,34 @@ type t
 
 (** {5 Constructors} *)
 
+(** Different ways to create a new universe state *)
+
 val empty : t
```

Comment:
IIRC `empty` is plain wrong. One should always use `from_env`.
While you are at it, you could write "don't use!" here.

Reply:
Good point.